pkg/kubeletconfig/config.go

```diff
@@ -0,0 +1,53 @@
+package kubeletconfig
+
+import (
+	"fmt"
+
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+
+	"github.com/openshift/rosa/pkg/interactive"
+)
+
+//go:generate mockgen -source=config.go -package=kubeletconfig -destination=mock_capability_checker.go
+type CapabilityChecker interface {
+	IsCapabilityEnabled(capability string) (bool, error)
+}
+
+// GetMaxPidsLimit - returns the maximum pids limit for the current organization
+// the maximum is varied depending on whether the current organizaton has
+// the capability.organization.bypass_pids_limit capability
+func GetMaxPidsLimit(client CapabilityChecker) (int, error) {
+	enabled, err := client.IsCapabilityEnabled(ByPassPidsLimitCapability)
```

Comment:
We usually leave the capability check to CS. I do see some UX improvements having it under CLI for this scenario. Just not sure... I also see there's a different check for capability elsewhere. Wouldn't block the PR due to this

Reply:
@gdbranco I think ideally we'd have an API on CS that allows the CLI to retrieve the min/max for `--pod-pids-limit` (and allowable values for other fields from `KubeletConfig` as we expose them), but that's beyond the scope of this ticket and XCMSTRAT-110 tbh. 

My feeling this is probably a reasonable compromise until we have that API because I can see feedback from customers that are allowed to exceed the limits such as "I thought we could go beyond 16,384..why does the CLI tell  me we can't?"

WDYT?

cheers,

Rob

Reply:
@gdbranco And for clarity: CS checks the capability when validating the submitted ranges. This is really for customer UX when using the CLI in terms of validation and error messaging.

Reply:
I think its okay to use capability checks in the CLI, its definitely feature flags we want to avoid in the CLI as the life span of a capability is generally life long, where as a feature flag is meant to be temp

I agree, we need  to move away and allow config be pulled from the API.